### PR TITLE
Implement home directory creation for consul user

### PR DESCRIPTION
**Pull request by sun77**
Poise-service doesn't create home directory with poise_service_user resource
Also allow us to change the home directory with an attribute

### CODE REVIEW

File: recipes/default.rb [hunk 1]
```diff
@@ -6,8 +6,19 @@
 #
 node.default['nssm']['install_location'] = '%WINDIR%'
 
+consul_home = node['consul']['service_user_home']
```
Comment (legal90) [inline recipes/default.rb]: I see that we use this variable only once. May be, we can omit that and just use an attribute as is?

**Comment by legal90** [comment]
@Ginja @johnbellone What do you think about this? Should we manage the home dir in this cookbook by default? 
I'm not sure about using `directory` resource for that. I would roll `poise_service_user` back to [`user` + `group` resources](https://github.com/johnbellone/consul-cookbook/blob/v2.1.3/recipes/default.rb#L36-L46) with setting `manage_home true`. Then the home dir will be created by "useradd" with system-defined attributes and permissions.

**Comment by Ginja** [comment]
https://github.com/poise/poise-service/issues/9

Why does the Consul user need a home? I'm in agreeance with @coderanger on this one. Generally service users don't have homes for security reasons. If one is required, I would put that logic in a wrapper cookbook.

**Comment by kamaradclimber** [comment]
consul user is the one used when launching health checks of type `script`. Many scripts/binary assume the existence of a $HOME

**Comment by johnbellone** [comment]
@kamaradclimber What would be an example?

**Pull request closed without merging**